sources/packages/backend/libs/sims-db/src/entities/msfaa-number.model.ts

```diff
@@ -103,6 +103,11 @@ export class MSFAANumber extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   student: Student;
+  /**
+   * ApplicationId related to this MSFAA.
+   */
+  @RelationId((msfaaNumber: MSFAANumber) => msfaaNumber.referenceApplication)
+  referenceApplicationId: number;
```

Comment:
Is it this being used anywhere, if not please remove it.
Was there any output about the idea of having this relation ids loaded in a different way?